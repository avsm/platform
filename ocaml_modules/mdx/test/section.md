## Title

### Hello

```ocaml
let x = 42 in
Printf.printf "%d\n%!" x
```

```
raw contents
```

### Toto

```ocaml
let x = 2
```

### Testing cram

```sh
 $ for i in `seq 1 10`; do echo $i; done
```

### Not testing cram

```sh
$ ls
foo bar
```

```ocaml
let () = print_endline (string_of_int x)
```

## Toplevel

```ocaml
   # print_endline "42";;
   42
```
