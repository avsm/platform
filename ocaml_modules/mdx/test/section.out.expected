42
2
42
